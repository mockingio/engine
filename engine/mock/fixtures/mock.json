{
  "name": "Hello World",
  "id": 123,
  "port": "3000",
  "routes": [
    {
      "method": "GET",
      "path": "/hello/world",
      "responses": [
        {
          "status": 200,
          "headers": {
            "Content-Type": "application/json"
          },
          "body": "{\"name\": \"John Doe\"}",
          "rule_aggregation": "and",
          "rules": [
            {
              "target": "",
              "value": "",
              "operator": ""
            }
          ]
        }
      ]
    },
    {
      "method": "GET",
      "path": "/greeting/world",
      "responses": [
        {
          "status": 200,
          "delay": 3000,
          "headers": {
            "Content-Type": "application/json"
          },
          "body": "{\"name\": \"Hi John Doe\"}",
          "rule_aggregation": "and",
          "rules": [
            {
              "target": "",
              "value": "",
              "operator": ""
            }
          ]
        }
      ]
    },
    {
      "method": "GET",
      "path": "/hello/*",
      "responses": [
        {
          "status": 200,
          "headers": {
            "Content-Type": "application/json"
          },
          "body": "{\"name\": \"John Doe\"}"
        }
      ]
    },
    {
      "method": "POST",
      "path": "/hello/world",
      "responses": [
        {
          "status": 201
        }
      ]
    }
  ]
}