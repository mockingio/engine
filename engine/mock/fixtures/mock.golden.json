{
  "id": "123",
  "name": "Hello World",
  "port": "3000",
  "routes": [
    {
      "request": "GET /hello/world",
      "description": "",
      "responses": [
        {
          "status": 200,
          "headers": {
            "Content-Type": "application/json"
          },
          "body": "{\"name\": \"John Doe\"}",
          "rule_aggregation": "and",
          "rules": [
            {
              "target": "",
              "value": "",
              "operator": ""
            }
          ]
        }
      ]
    },
    {
      "request": "GET /greeting/world",
      "description": "",
      "responses": [
        {
          "status": 200,
          "delay": 3000,
          "headers": {
            "Content-Type": "application/json"
          },
          "body": "{\"name\": \"Hi John Doe\"}",
          "rule_aggregation": "and",
          "rules": [
            {
              "target": "",
              "value": "",
              "operator": ""
            }
          ]
        }
      ]
    },
    {
      "request": "GET /hello/*",
      "description": "",
      "responses": [
        {
          "status": 200,
          "headers": {
            "Content-Type": "application/json"
          },
          "body": "{\"name\": \"John Doe\"}"
        }
      ]
    },
    {
      "request": "POST /hello/world",
      "description": "",
      "responses": [
        {
          "status": 201
        }
      ]
    }
  ]
}